eva_submission/eload_validation.py

```diff
@@ -432,64 +440,35 @@ def _collect_normalisation_check_results(self, vcf_files, output_dir):
             })
             total_error += len(error_list)
         self.eload_cfg.set('validation', 'normalisation_check', 'pass', value=total_error == 0)
-
-    def _detect_structural_variant(self):
-        vcf_files = self._get_vcf_files()
+    def _collect_structural_variant_check_results(self, vcf_files, output_dir):
+        # detect output files for structural variant check
+        total_error = 0
         for vcf_file in vcf_files:
-            has_sv_per_vcf = self._detect_structural_variant_from_variant_lines(vcf_file)
-            if not has_sv_per_vcf:
-                vcf_name = os.path.basename(vcf_file)
-                validator_output_file = os.path.join(self._get_dir('vcf_check'), vcf_name + '.vcf_validator.txt')
-                has_sv_per_vcf = self._detect_structural_variant_from_validator_output(validator_output_file)
+            vcf_name = os.path.basename(vcf_file)
+
+            tmp_sv_check_log = resolve_single_file_path(
+                os.path.join(output_dir, 'sv_check',  vcf_name + '.sv_check.log')
+            )
+            tmp_sv_check_sv_vcf = resolve_single_file_path(
+                os.path.join(output_dir, 'sv_check', vcf_name + '.sv_list.vcf')
```

Comment:
You have `${vcf_file.getSimpleName()}_sv_list.vcf` in the nextflow rather than `.sv_list.vcf` (and same for `.sv_check.log`...)